src/libs/actions/IOU.ts

```diff
@@ -10743,6 +10748,285 @@ function getSearchOnyxUpdate({participant, transaction}: GetSearchOnyxUpdatePara
     }
 }
 
+/**
+ * Initialize split expense info
```

Comment:
Maybe we could improve the comment there. What does it do specifically? Why is it needed?